src/measure_theory/measure_space.lean

```diff
@@ -1444,8 +1436,37 @@ instance probability_measure.to_finite_measure (μ : measure α) [probability_me
 lemma probability_measure.ne_zero (μ : measure α) [probability_measure μ] : μ ≠ 0 :=
 mt measure_univ_eq_zero.2 $ by simp [measure_univ]
 
+lemma prob_add_compl_eq_one [probability_measure μ]
+  (h: measurable_set s) : μ s + μ sᶜ = 1 :=
+begin
+  rw measure_add_measure_compl,
+  rw measure_univ,
+  exact h,
+end
+
+lemma prob_le_one [probability_measure μ] : μ s ≤ 1 :=
+begin
+  have h₁ : μ s ≤ μ set.univ := measure_mono (set.subset_univ _),
+    rw measure_univ at h₁,
+  exact h₁,
```

Comment:
```suggestion
  rwa measure_univ at h₁,
```
No indentation needed for `rw`, it is just following the normal stream of the proof. Also, `rwa` tries to close the goal by using one of the statements after the rewrite, so it works perfectly fine here.